src/button/Button.tsx

```diff
@@ -237,3 +238,20 @@ export function InviteButton({
     </TooltipTrigger>
   );
 }
+
+export function OptionsButton({
+  className,
+  ...rest
+}: {
+  className?: string;
+  [index: string]: unknown;
```

Comment:
Is it possible to make this any more specific? Such as `Omit<"variant", ButtonProps>` or something

Reply:
Or rather, turn the type into an interface that extends ButtonProps